### DISCUSSION TITLE
Beginner Friendly Mode

### DISCUSSION THREAD

**Discussion by annasassen** [phcode-dev/phoenix#1992] [Ideas]
I really like the look and feel of Phoenix Code and I'd like to continue using it, but I constantly feel like I am fighting with it. The (extremely difficult to disable) auto complete features are nice, but as a coding newbie I was highly encouraged to build the muscle memory of closing tags and not to rely on the auto closing/completing features that so many editors have these days. I understand that they are useful and speed up coding. Perhaps a good way to help Phoenix Code grow would be to give it a VERY beginner friendly "mode". Having a mode (and a great tutorial on how to use Phoenix Code) for beginners and partnering with sites like freeCodeCamp or W3Schools, etc., might be advantageous. n00bs like me would likely be drawn to a beginner friendly editor and I'd imagine they'd stick with it, because it IS a really good editor. I've tried like 7 or 8 different editors looking for one that will work for me as a beginner, and I have had no end of frustration and it's made me want to give up! But, Phoenix Code is my favorite by far, pesky (to a newbie but useful to someone with experience) auto complete settings, aside. Giving Phoenix Code a beginner friendly option would certainly set it apart from the rest and considering the projections for growth in web design and development, it could be really worth it!

**Comment by abose** [comment]
Thank you so much for your detailed feedback and for sharing your experience with Phoenix Code. It’s incredibly valuable to us and aligns with our goal of creating an editor that’s both beginner-friendly and powerful enough to grow with you as your skills develop.

We designed the default experience of Phoenix Code with beginners in mind, aiming to strike a balance between simplicity and capability. Our goal was to create a tool that helps you focus on learning and coding effectively, without needing to toggle between multiple modes. From our research and feedback from users worldwide, we've found that assistive features, such as auto-completion, can significantly improve the learning curve and ramp-up time for most students. This is why Phoenix Code incorporates assistive technologies that we believe can enhance productivity and reduce frustration over time.

That said, we deeply appreciate your perspective about building muscle memory and how certain features might feel overwhelming when starting out. Your suggestion for a “beginner mode” or more granular control over assistive settings is insightful. For instance:

1. **Better settings UX to Disable Auto-complete**: Providing easier access to toggle settings like preferences is tracked in https://github.com/phcode-dev/phoenix/issues/147
2. **Interactive Tutorials**: Maybe a hands on training mode for beginners may help. we are tracking this as part of an earlier feature request https://github.com/phcode-dev/phoenix/issues/1013 

Your idea about partnering with educational platforms like freeCodeCamp or W3Schools is also an excellent suggestion. We’re exploring ways to integrate tutorials and workflows that complement learning paths from these resources. We have a dedicated youtube channel at https://www.youtube.com/@phoenixcodeeditor3764 which we may add intro learning resources in the future.

We’re committed to evolving Phoenix Code based on user feedback like yours. Rest assured, we’ll consider how to make workflows more beginner-friendly while maintaining the power and flexibility that make Phoenix Code a great editor for all skill levels. Thank you for helping us improve!
